xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -398,13 +398,13 @@ bool CPVRTimerInfoTag::UpdateOnClient()
 void CPVRTimerInfoTag::DisplayError(PVR_ERROR err) const
 {
   if (err == PVR_ERROR_SERVER_ERROR)
-    CGUIDialogOK::ShowAndGetInput(19033,19111,19110,0); /* print info dialog "Server error!" */
+    CGUIDialogOK::ShowAndGetInput(19033, 19111, -1, -1); /* print info dialog "Server error!" */
```

Comment:
you can drop the last two parameters `-1, -1` and use the method `CGUIDialogOK::ShowAndGetInput(const CVariant &heading, const CVariant &text)`